packages/firefox/firefox.pacscript

```diff
@@ -0,0 +1,24 @@
+name="firefox"
+version="92.0.1"
+maintainer="aranym <aranym1@pm.me>"
+url="https://download-installer.cdn.mozilla.net/pub/${name}/releases/${version}/linux-x86_64/en-US/${name}-${version}.tar.bz2"
+depends="lsb-release libatk1.0-0 libc6 libcairo-gobject2 libcairo2 libdbus-1-3 libdbus-glib-1-2 libfontconfig1 libfreetype6 libgcc-s1 libgdk-pixbuf-2.0-0 libglib2.0-0 libgtk-3-0 libharfbuzz0b libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb-shm0 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrender1"
+description="Safe and easy web browser from Mozilla"
+hash="a3ba62122a6cde1d7d2ebf96e765ea57ea3e9a312d2161e0ac70923946844689"
+
+prepare() {
+    sudo mkdir -p /usr/src/pacstall/firefox/usr/share/icons
+    sudo mkdir -p /usr/src/pacstall/firefox/usr/bin
+    sudo mkdir -p /usr/src/pacstall/firefox/usr/share/applications
+    wget -q https://gist.githubusercontent.com/aranym0/dad87fbc9f388e2917d625534e8f05a5/raw/e4e8014ed7e2cf1b05971e048074f0ba7e9b5845/firefox.desktop 
+}
+
+build() {
+    true
+}
+
+install() {
+    sudo mv ./browser/chrome/icons/default/default128.png /usr/src/pacstall/firefox/usr/share/icons
+    sudo mv ./firefox.desktop /usr/src/pacstall/firefox/usr/share/applications
+    sudo mv ./* /usr/src/pacstall/firefox/usr/bin
```

Comment:
Your moving every thing to `/usr/bin`?

```
Throbber-small.gif  firefox          libfreeblpriv3.so  libmozwayland.so  librlbox.so        omni.ja               update-settings.ini
application.ini     firefox-bin      liblgpllibs.so     libnspr4.so       libsmime3.so       pingsender            updater
browser             firefox-bin.sig  libmozavcodec.so   libnss3.so        libsoftokn3.so     platform.ini          updater.ini
crashreporter       firefox.sig      libmozavutil.so    libnssckbi.so     libssl3.so         plugin-container
crashreporter.ini   fonts            libmozgtk.so       libnssutil3.so    libxul.so          plugin-container.sig
defaults            gmp-clearkey     libmozsandbox.so   libplc4.so        libxul.so.sig      precomplete
dependentlibs.list  icons            libmozsqlite3.so   libplds4.so       minidump-analyzer  removed-files
```

That's the full list of stuff included in the tar. Your moving all that... What.

Reply:
```
    sudo install -D ./* /usr/src/pacstall/firefox-bin/opt/firefox-bin
    sudo ln -s "$STOWDIR/$name/opt/firefox-bin/firefox" "/usr/bin/firefox"
```

Do you think something like this would be a reasonable solution to that issue?

Reply:
Why move firefox to `opt`? I'm not getting what you mean.

> I would move all of the icons and desktop file beforehand, so no unnecessary junk would be copied over.

All the `lib*` has to be moved to the correct location too. Plus the `updater` and `updater.ini` are for Windows... They don't work in linux.

See the PKGBUILD for guidance on what to move.